wire-ios/Wire-iOS/Sources/Helpers/Bundle/Bundle+WireURLs.swift

```diff
@@ -90,6 +90,9 @@ struct WireURLs: Codable {
     /// Link to an article explaining how to add a conversation to your favourites folder.
     let howToAddConversationToYourFavourites: URL
 
+    /// Link to an article explaining how to add a conversation to a custom folder.
+    let howToAddAConversationToACustomFolder: URL
```

Comment:
Please don't forget to make the adjustment in the build assets repo. 
See this PR: https://github.com/wireapp/wire-ios-build-assets/pull/128/files

Reply:
I've created a followup ticket to do this for this sprint: https://wearezeta.atlassian.net/browse/WPB-14336. Thank you I wouldn't have thought of this!